metricbeat/module/beat/stats/data_test.go

```diff
@@ -33,7 +33,7 @@ import (
 
 func TestEventMapping(t *testing.T) {
 
-	files, err := filepath.Glob("./_meta/test/stats.*.json")
+	files, err := filepath.Glob("./_meta/test/*.stats.*.json")
```

Comment:
[to reviewer] it wasn't testing apm-server.stats.712.json, not sure if it was intended.

Reply:
no idea but we'll know if CI fails :)